src/cli/tc.rs

```diff
@@ -42,6 +42,9 @@ pub fn run() -> Result<()> {
           if schema {
             display_schema(&blueprint);
           }
+          if let Some(out_file) = out_file_name {
+            write_output_file(&out_file, &config)?;
```

Comment:
Like `config::Reader` we should add `config::Writer` that can write the file.

Reply:
Hi @tusharmath, added config writer. I hope this is what you were expecting?